### ISSUE TITLE
Make configuration easier to see/edit

### ISSUE THREAD

**Issue by fritzophrenic** [fireout/keepasssequencer#46]
It was good to get rid of the option dump in each sequence item, I think the new interface is a lot clearer.

But there's still a need to see current settings. And editing through free-form text field is not very user friendly.

Let's make a custom dialog for each configuration item that isn't a simple checkbox.

**Capitalization** could have radio buttons for "Always", "Never", "Proper", and "Randomly". Next to "Randomly" should be a numeric edit with up/down buttons.

**Probability** and **Substitution** would just have a numeric edit with up/down buttons. Note this would let us set maximums and minimums more easily to prevent the user entering 500% or something manually.

I'd get rid of **Length Strength** entirely, combining it with Length:

**Length** should be a numeric edit with up/down buttons, defaulting to 1 character. If Advanced Mode is active, there would also be a "Chance of reaching full length" numeric field that defaults to 100%.

If it's possible, it would also be nice to put some sort of readout/preview on the drop-down configuration menu. Something like this:

![char_config](https://cloud.githubusercontent.com/assets/1334420/13726049/590917f4-e87b-11e5-9a0f-026dfe019610.png)
![word_config](https://cloud.githubusercontent.com/assets/1334420/13726050/5bbc6136-e87b-11e5-9463-c53abe9a5eb4.png)

Note how Length Strength is combined with Length. For Capitalization, I'm envisioning showing one of the following: "ABC" for always, "abc" for never, "aBc" for any random percentage, and "Abc" for proper.

Feedback welcome before implementation.

**Comment by fireout** [comment]
I'm a little hesitant displaying the output/readout in the menu, mainly because of the room available...  The suggested legend works good for now, maybe we should also display readout when hovering on sequence item?  Something like:

---

100% Probability
3 Characters long 
Include default characters

---

or

---

 _50% Probability_ 
Proper capitalization
Exclude default words

---

it could also display settings more explicitly...  what do you think?

**Comment by fritzophrenic** [comment]
I like that idea a lot, good thinking! I'd still probably combine length and length strength though.

**Comment by fireout** [comment]
I agree, we could have the same mentionned 0-x character for when it's not 100% strong.

It also simplifies the interface, I like it.
